README.md

```diff
@@ -14,6 +14,37 @@ following actions are implemented
     $ git clone https://github.com/cognoma/figshare.git
     $ python setup.py install
 
+For development, I recommend:
+
+    $ python setup.py develop
+
+## Execution
+
+To use this Python module, here's a demonstration of initiating the API and
+retrieving information from Figshare API. This was done using
+[iPython](https://ipython.readthedocs.io/en/stable/), but
+other Python-based methods will work as well:
+
+    In [1]: from figshare.figshare import Figshare
+    In [2]: fs = Figshare()
+    In [3]: article_id = 4757131
+    In [4]: fs.get_article_details(article_id)
```

Comment:
It's probably best to remove `In [1]: ` from each line, such that users can directly copy and execute the entire code block.

Also consider using triple backticks for the codeblock, which I find easier (I know elsewhere in the README we use indentation for codeblocks).